frontend/src/pages/projects/components/PVSizeField.tsx

```diff
@@ -60,8 +60,9 @@ const PVSizeField: React.FC<PVSizeFieldProps> = ({
                 variant="warning"
                 icon={<ExclamationTriangleIcon />}
               >
-                Storage size can only be increased. If you do so, the workbench will restart and be
-                unavailable for a period of time that is usually proportional to the size change.
+                {currentStatus.conditions?.find((c) => c.type === 'FileSystemResizePending')
```

Comment:
this status will not show up hen they are adjusting the size.  the condition will only show once the change has been made. If we want it in here we need to instead check if it is not connected to any running notebooks, then show the warning.

Reply:
This needs configuring BaseStorageModal and ClusterStorageModal, which i'll do when [this](https://github.com/opendatahub-io/odh-dashboard/pull/3529) PR gets merged